libraries/ms-common/src/can.c

```diff
@@ -145,14 +152,17 @@ bool can_process_event(const Event *e) {
 }
 
 void prv_tx_handler(void *context) {
-  CanStorage *can_storage = context;
-  CanMessage tx_msg;
-
-  // If we failed to TX some messages or aren't transmitting fast enough, those
-  // events were discarded. Raise a TX event to trigger a transmit attempt. We
-  // only raise one event since TX ready interrupts are 1-to-1.
-  if (can_fifo_size(&can_storage->tx_fifo) > 0) {
-    event_raise(can_storage->tx_event, 0);
+  // following condition used to disable tx events being re-raised on x86
+  // as this causes a race condition: see soft_301_race_condition_x86_can
```

Comment:
By the ticket I did mean the Jira ticket designation like `SOFT-301` (`soft_301_race_condition_x86_can` is the branch, which will be deleted from github after the PR is merged, so it won't be useful to future devs)